components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -983,17 +986,19 @@ private Response handleAuthenticationResponse(OAuthMessage oAuthMessage)
             throws OAuthSystemException, URISyntaxException, ConsentHandlingFailedException {
 
         if (LoggerUtils.isDiagnosticLogsEnabled()) {
-            Map<String, Object> requestParams = new HashMap<>();
+            DiagnosticLog.DiagnosticLogBuilder diagnosticLogBuilder = new DiagnosticLog.DiagnosticLogBuilder(
+                    OAuthConstants.LogConstants.OAUTH_INBOUND_SERVICE, "receive-authn-response");
```

Comment:
Shall we use a constant here

Reply:
Fixed with f885f3117702fe7ecf5cadc66a35c3f3eea4bddd